### DISCUSSION TITLE
12.27题目答案纠错/讨论

### DISCUSSION THREAD

**Discussion by luminosara** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#27] [General]
> 1508241 以下说法不正确的有 【多选题】
> 4948295 Redmi所有手机均可使用EDL模式
> ✔️ 选中。仅高通（Qualcomm）平台的手机可以通过官方售后使用 Qualcomm Emergency Download（EDL, 9008 模式）刷机。
> 4948296 Android系统启动时，首先启动的是zygote进程
> ❌ 不选。Zygote 进程由 Android 运行时（Android Runtime, ART）启动。所以第一个启动的进程并不是 Zygote。
> 4948297 ROM指的是手机的运行内存
> ✔️ 选中。ROM 全称为 Read-only memory，指的是手机的系统固件。运行内存为 Random-access memory（RAM）。
> 4948298 特征码查杀技术可以有效检测采用加壳技术的恶意程序
> ✔️ 选中。恶意软件加壳（或加密/加固）后，常规的特征会被外壳覆盖，使特征码查杀技术失效。
> 4948299 adb的全称是Android Debug Badge，起到调试桥的作用
> ❌ 不选。该描述是正确的。

题干中选择的是错误的选项，`4948296 Android系统启动时，首先启动的是zygote进程`应该是错误的，因而应该选中？见前几期同样的选项为选中。

（不过这几个星期答题同样只有83分...不知道哪几个选项mi认为是错的了...

**Comment by MlgmXyysd** [comment]
对，这个描述错的，脚本规则匹配出问题了，我改一下

有一题存疑的选项可能答案不对，需要5级号来测试一下

**Reply by luminosara** [reply]
感谢感谢！

**Reply by MlgmXyysd** [reply]
18点以后答不了

**Comment by ehJunyeah** [comment]


**Reply by MlgmXyysd** [reply]
答案有错的，不知道哪个错了🤔

**Comment by m1w22** [comment]
今天答了75分，错了5道，还要截图验证

还有1508230是对的吧

https://source.android.com/docs/core/architecture/bootloader?hl=zh-cn

4948259也是对的吧，inetd是inet守护进程

https://www.ibm.com/docs/en/ibm-mq/9.1?topic=linux-establishing-tcp-connection

下周答题试试，看看是不是对的

**Reply by dcjx** [reply]
那4948249应该也是对的吧

**Reply by MlgmXyysd** [reply]
inet**d**是重要的网络守护进程，但是题目原文是inet，没有d
1508230这题以前能验证答案的时候出过，错误的
可以传一下录屏或截图，我帮着看下

**Reply by MlgmXyysd** [reply]
小米今天改正了，加了个d，这下就确定它是正确的了
![image](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/17919858/5a727f3f-52ca-45ce-a692-a605dc9c373b)
